docker/images/python/Dockerfile.dev

```diff
@@ -1,10 +1,10 @@
 FROM python:slim
 
-RUN apt-get update \
-    && apt-get install --no-install-recommends --no-install-suggests -y \
-    make wkhtmltopdf xvfb gettext
-    && apt-get -y --purge autoremove \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
```

Comment:
moving `&&` to line end is not needed code noise.
Now we do `&&` at the beginning of the line at all files.
We also move operators at the beginning of the line in python:
```
strr = (
    'one'
    + 'two'
    + 'three'
)
```